core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -560,9 +672,10 @@ private[spark] class ExecutorAllocationManager(
       val stageAttemptId = stageCompleted.stageInfo.attemptNumber()
       val stageAttempt = StageAttempt(stageId, stageAttemptId)
       allocationManager.synchronized {
-        // do NOT remove stageAttempt from stageAttemptToNumRunningTasks,
+        // do NOT remove stageAttempt from stageAttemptToNumRunningTask
```

Comment:
nit: Do we need to update this line?

Reply:
No but the comment was wrong there is no stageAttemptToNumRunningTasks datastructure and didn't want it confusing for people reviewing the change, I can remove if you like?